R/calculate_tdm.R

```diff
@@ -0,0 +1,305 @@
+#' Calculate and add the transition disruption metric to a dataset
+#'
+#' This function takes a typical PACTA output, and uses it to calculate the
+#' transition disruption metric.
+#'
+#' @param data A dataset, like the `*_results_portfolio.rda` outputs of PACTA.
+#' @param t0 The start year, from which the TDM value should be calculated.
```

Comment:
can you suggest what would be a reasonable setting for this?

Reply:
Personally, I don't think we should set a default argument for this parameter (since we would have to update it every year). 

Or do you mean to explain further what "start year" means?

Reply:
yeah, like just an explanation of what it usually would be (relative to the given dataset).... like "this would usually be the earliest year in your PACTA results dataset"